cclib/io/cjsonwriter.py

```diff
@@ -77,7 +77,9 @@ def as_dict(self):
             # obtained from the output file. TODO rewrite this
             if attribute_name in ('moments', 'atomcoords'):
                 if attribute_name == 'moments':
-                    cjson_dict['properties'][ccData._attributes['moments'].json_key] = self._calculate_total_dipole_moment()
+                    dipole_moment = self._calculate_total_dipole_moment()
+                    if dipole_moment is not None:
+                        cjson_dict['properties'][ccData._attributes['moments'].json_key] = self._calculate_total_dipole_moment()
```

Comment:
Any reason for the second call of `_calculate_total_dipole_moment()` instead of just using `dipole_moment`?

Reply:
Nope! Done.